bin/miden-tx-prover/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+name = "miden-tx-prover"
+version = "0.6.0"
+description = "Miden rollup transaction prover"
+readme = "README.md"
+categories = ["no-std"]
+keywords = ["miden", "transaction"]
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+
+[[bin]]
+name = "miden-tx-prover-service"
+path = "src/bin.rs"
+
+[features]
+default = ["std"]
+std = []
+testing = ["miden-objects/testing", "miden-lib/testing", "miden-tx/testing"]
+async = []
```

Comment:
Why do we need the `async` feature here? Could we not always import `miden-tx` with `async` feature enabled?